framework/src/test/java/io/github/kgress/scaffold/webdrivercontext/ExplicitWaitTests.java

```diff
@@ -0,0 +1,23 @@
+package io.github.kgress.scaffold.webdrivercontext;
+
+import io.github.kgress.scaffold.BaseUnitTest;
+import io.github.kgress.scaffold.webdriver.TestContext;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.github.kgress.scaffold.webdriver.interfaces.TestContextSetting.WAIT_FOR_DISPLAY_ENABLED;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ExplicitWaitTests extends BaseUnitTest {
+
+    @Before
+    public void enableExplicitWaits() {
+        TestContext.baseContext().addSetting(WAIT_FOR_DISPLAY_ENABLED, true);
```

Comment:
would it be worth to include a debug log statement here?

Reply:
I don't think so. I personally don't do much logging for unit testing. Do you think there's value here since it's a `@Before` method?

Reply:
oh, good point. within the scope of a unit test, yeah not needed.